.travis.yml

```diff
@@ -1,11 +1,14 @@
 language: python
 python:
   - '3.6'
+before_install:
+  - python --version
+  - pip install -U pip
+  - pip install -U pytest
```

Comment:
we don't want to lock the pytest version in requirements.txt?

Reply:
We could lock the pytest version, but we will have to update it frequently since travis will fail when new versions are released ([source](https://blog.travis-ci.com/2019-08-07-extensive-python-testing-on-travis-ci))